src/components/Card/Card.js

```diff
@@ -0,0 +1,44 @@
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+Card.propTypes = {
+  images: PropTypes.node.isRequired,
+  title: PropTypes.string,
+  text: PropTypes.string,
+}
+
+export default function Card({ text, title, images }) {
```

Comment:
I'd name it `image` as it is only one.
```suggestion
export default function Card({ text, title, image }) {
```